intermediate_oop/tic_tac_toe/js/main.js

```diff
@@ -1,2 +1,47 @@
 document.addEventListener("DOMContentLoaded", function() {
+	let game = new Grid();
+	let $square = $(".square-set");
+	let $result = $(".result");
+	let $reset = $("#new-game")
+
+	//Move
+	$square.on("click",function(event) {
+		if($(event.target).text() === "" && game.on) {
+			if (game.move % 2) {
+				$(event.target).text("O");
+				game.addResult("O",$(event.target)[0].id);
+				game.increaseMove();
+				if (game.testWin("O")) {
+					game.createPage($result, "win", "O");
+					$square.removeClass("square-set");
+					game.turnOff();
+				} else if (game.move === 9) {
+					game.createPage($result, "tie");
+					$square.removeClass("square-set");
+					game.turnOff();
+				}
+			} else {
+				$(event.target).text("X");
```

Comment:
the code in here is very similar to the code inside of the if block, just for "X" instead of "O." Can you think of a way to refactor to reduce duplication?